eva-accession-release/src/main/resources/copy_release_to_ftp.sh

```diff
@@ -188,7 +204,7 @@ do
   taxonomy=`echo "${assembly_and_species}" | cut -f 2`
 
   assembly_folder=${OUTPUT_FOLDER}/${ASSEMBLIES}/${assembly}
-  species_folder=`grep "        ${taxonomy}$" $INPUT_FOLDER/species_name_mapping.tsv | cut -f 1`
+  species_folder=`grep "${taxonomy}$" $INPUT_FOLDER/species_name_mapping.tsv | cut -f 1`
```

Comment:
these tabs were intentional to not match a subset of the number, say, not matching tax 123 in the line for 5123. applies to other lines

Reply:
Changed the grep command to use `-w` to match only whole words